index.html

```diff
@@ -0,0 +1,134 @@
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <li><button>See project</button></li>
+                </ul>
+            </li>
+            <li class="container2">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot4.png" alt=""></li>
+                    <li class="works-heading">Multi-Post Stories</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <li><button>See project</button></li>
+                </ul>
+            </li>
+        </ul>
+    </section>
+    <section id="about-self">
+        <h3 class="aboutself-heading">About myself</h3>
+        <p class="aboutself-paragragh">Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see  and have a project you need coded, don't hesitate to contact me.</p>
+        <h4 class="connect2">LET'S CONNECT</h4>
+            <ul id="social-icons">
+                <li><a href="#" class="icon"><img src="Assets/files/assets/github.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/Linkedin icon.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/Vector.jpg" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/twitter.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/medium.png" alt="Loading"></a></li>
+            </ul>
+        <button class="resume">Get my resume</button>
+
+        <ul class="langues">
+            <li class="languages1">Languages</li>
+            <div class="arrow"> <img src="Assets/files/arrow.png" alt=""></div>
+                <ul id="tech">
+                    <li><button class="tag tag1"> Javascript</button></li>
+                    <li><button class="tag tag2"> HTML</button></li>
+                    <li><button class="tag tag3"> CSS</button></li>    
+                </ul>
```

Comment:
- Good job done so far, to help improve the semantics of your page, only `li` tags should be direct children of the `ul` tag, so I suggest you put this section inside the first `li` tag i.e the `li` tag with class name `languages1`. Please fix this 😄